src/main/resources/templates/index.html

```diff
@@ -1,9 +1,14 @@
+<!--prettier-ignore-->
```

Comment:
If the intention is to ignore all files in `resources/templates` then it might be better to use a [prettierignore](https://prettier.io/docs/en/ignore.html) file instead of using `<!-- prettier-ignore -->` everywhere